docs/data-sources/compute_instance_list.md

```diff
@@ -37,15 +39,60 @@ output "my_compute_instance_ids" {
+- `disk_size` (Number) Match against this int
+- `id` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `ipv6` (Boolean) Match against this bool
+- `ipv6_address` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `labels` (Map of String) Match against key/values. Keys are matched exactly, while values may be matched as a regex if you supply a string that begins and ends with "/"
+- `manager_id` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `manager_type` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `name` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `public_ip_address` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `reverse_dns` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `ssh_key` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `state` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `template_id` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `type` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `user_data` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+- `zone` (String) Match against this string. If you supply a string that begins and ends with a "/" it will be matched as a regex.
+
+### Read-Only
+
+- `instances` (List of Object) The list of [exoscale_compute_instance](./compute_instance.md). (see [below for nested schema](#nestedatt--instances))
+
+<a id="nestedatt--instances"></a>
+### Nested Schema for `instances`
+
+Read-Only:
+
+- `anti_affinity_group_ids` (Set of String)
+- `created_at` (String)
+- `deploy_target_id` (String)
+- `disk_size` (Number)
+- `elastic_ip_ids` (Set of String)
+- `id` (String)
+- `ipv6` (Boolean)
+- `ipv6_address` (String)
+- `labels` (Map of String)
+- `manager_id` (String)
+- `manager_type` (String)
+- `name` (String)
+- `private_network_ids` (Set of String)
+- `public_ip_address` (String)
+- `reverse_dns` (String)
+- `security_group_ids` (Set of String)
+- `ssh_key` (String)
+- `state` (String)
+- `template_id` (String)
+- `type` (String)
+- `user_data` (String)
+- `zone` (String)
```

Comment:
Notice that for a nested list of objects the plugin generating the docs doesn't include the descriptions. This is unfortunately an issue we can't do much about:
https://github.com/hashicorp/terraform-plugin-docs/issues/28#issuecomment-768299611